bigchaindb/upsert_validator/validator_election_vote.py

```diff
@@ -0,0 +1,74 @@
+import base58
+
+from bigchaindb.common.transaction import Transaction
+from bigchaindb.common.schema import (_validate_schema,
+                                      TX_SCHEMA_COMMON,
+                                      TX_SCHEMA_TRANSFER,
+                                      TX_SCHEMA_VALIDATOR_ELECTION_VOTE)
+
+
+class ValidatorElectionVote(Transaction):
+
+    VALIDATOR_ELECTION_VOTE = 'VALIDATOR_ELECTION_VOTE'
+    # NOTE: this transaction class extends create so the operation inheritence is achieved
+    # by renaming CREATE to VALIDATOR_ELECTION
+    TRANSFER = VALIDATOR_ELECTION_VOTE
+    ALLOWED_OPERATIONS = (VALIDATOR_ELECTION_VOTE,)
+
+    def __init__(self, operation, asset, inputs, outputs,
+                 metadata=None, version=None, hash_id=None):
+        # operation `CREATE` is being passed as argument as `VALIDATOR_ELECTION` is an extension
```

Comment:
Bad copypaste detected :)

No empty `__init__` - no chance to make such a mistake.